package.json

```diff
@@ -4,8 +4,29 @@
   "description": "Calendar board for PMSIpilot organisation",
```

Comment:
dans tous les cas, le `"private": true` devrait être ajouté : c'est une application, pas un module donc aucune raison de la publier sur NPM.

Par contre, en effet, la description devrait peut-être être du genre `Calendar board for Google Apps organisations`